app/docs/enterprise/0.31-x/installation/amazon-linux.md

```diff
@@ -0,0 +1,81 @@
+---
+title: How to install Kong Enterprise and Postgres onto Amazon Linux
+---
+
+# How to Install Kong Enterprise and Postgres onto Amazon Linux
```

Comment:
suggestion: PostgreSQL - "Postgres" feels colloquial in the context of technical, enterprise-grade documentation.